cache-filter/src/filter/http.rs

```diff
@@ -58,7 +58,7 @@ enum AuthResponseError {
     ListKeysMiss,
     #[error("app id field is missing from list keys extension response")]
     ListAppIdMiss,
-    #[error("app id field is missing from list keys extension response")]
+    #[error("service id field is missing from list keys extension response")]
```

Comment:
change commit title to remove "make change to" - it is always implicit :)

Reply:
Oh Yea :sweat_smile: